snuba/subscriptions/codecs.py

```diff
@@ -97,14 +96,14 @@ def decode(self, value: KafkaPayload) -> ScheduledSubscriptionTask:
 
         return ScheduledSubscriptionTask(
             datetime.fromisoformat(scheduled_subscription_dict["timestamp"]),
-            SubscriptionWithTick(
+            SubscriptionWithMetadata(
                 entity_key,
                 Subscription(
                     SubscriptionIdentifier.from_string(subscription_identifier),
                     SnQLSubscriptionData.from_dict(
                         scheduled_subscription_dict["task"]["data"], entity_key
                     ),
                 ),
-                Tick.from_dict(scheduled_subscription_dict["tick"]),
+                scheduled_subscription_dict["tick_upper_offset"],
```

Comment:
This is push safe because the executor is not yet running right ?

Reply:
Yes